Explorer++/Explorer++/Storage.cpp

```diff
@@ -10,8 +10,31 @@
 namespace Storage
 {
 
+std::optional<std::wstring> GetEnvironmentVariableWrapper(const std::wstring &name)
+{
+	auto length = GetEnvironmentVariable(name.c_str(), nullptr, 0);
+	if (length == 0)
+	{
+		return std::nullopt;
+	}
+
+	std::wstring value;
+	value.resize(length);
+	length = GetEnvironmentVariable(name.c_str(), value.data(), length);
+	if (length == 0)
+	{
+		return std::nullopt;
+	}
+	return value;
+}
+
 std::wstring GetConfigFilePath()
 {
+	auto configPath = GetEnvironmentVariableWrapper(CONFIG_FILE_ENV_VAR_NAME);
+	if (configPath) {

```

Comment:
The opening brace here should appear on the next line. Could you make sure that you format the code using ClangFormat. Visual Studio has [built-in support](https://learn.microsoft.com/en-us/visualstudio/ide/reference/options-text-editor-c-cpp-formatting?view=vs-2022) for ClangFormat, so you should be able to format from within the IDE.